### PR TITLE
fix: Cleaning up AI Monitoring quickstarts

### PR DESCRIPTION
**Pull request by RyanKadri**
# Summary

This PR cleans up a couple AI Monitoring quickstarts:
- Removes or fixes old/misleading alert conditions for LangChain and OpenAI
- Point the Azure OpenAI quickstart to the correct installation wizard
- Point the LangChain quickstart to the correct installation wizard and removes an outdated dashboard in favor of curated AIM landing pages.

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you include a Data source and Documentation reference?
- [x] Are all documentation links publicly accessible?
- [x] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [x] Did you check your descriptive content for spelling and grammar errors?
- [x] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [x] Does the PR contain a screenshot for each of your dashboards?
- [x] Do your screenshots show data?
- [x] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [x] Did you check that your alerts actually work?

### CODE REVIEW

File: quickstarts/mlops/langchain/config.yml [hunk 1]
```diff
@@ -59,5 +59,4 @@ keywords:
   - orchestration
 alertPolicies:
   - langchain
-dashboards:
-  - langchain
+dashboards: []
```
Comment (RamanaReddy8801) [inline quickstarts/mlops/langchain/config.yml:62]:
```suggestion

```

File: alert-policies/langchain/responseTime.yml [hunk 2]
```diff
@@ -4,7 +4,7 @@ description: |+
   This alert is triggered if response time exceeds 5 seconds in 5 minutes.
```
Comment (RamanaReddy8801) [inline alert-policies/langchain/responseTime.yml:4]:
```suggestion
  This alert is triggered if duration exceeds 5 seconds in 5 minutes.
```

**Review by RamanaReddy8801 — CHANGES_REQUESTED** [review]
@RyanKadri, please make requested changes and change the alert file name `responseTime.yml` to `duration.yml`

**Comment by RyanKadri** [comment]
@RamanaReddy8801 I made those changes. How do things look now? Thanks!

**Review by sjyothi54 — APPROVED** [review]
Looks good

**Review by RamanaReddy8801 — APPROVED** [review]
LGTM

**Pull request merged**